web/src/app/common/services/rest.service.ts

```diff
@@ -30,34 +32,24 @@ export class RestService {
             .catch(this.handleError);
     }
 
-    public claimTokens() {
+    public getRewardHistory() {
         const headers = new HttpHeaders().set('Content-Type', 'application/json').set('authorization', 'Eevoo0aemah1ohY6Oheehee4ivahR5ae');
-        const url = '/rwd/multisig/6/sporwc';
+        const url = '/rwdinfo/rewards/history/' + globalThis.wallet.sending_stake_addr;
         RestService.processingRequest = true;
 
         return lastValueFrom(this.httpClient
-            .post(url, globalThis.wallet, {headers: headers}))
+            .get(url, {headers: headers}))
             .then(res => this.processResponse(res))
             .catch(this.handleError);
-    }
 
-    public fakeClaimTokens(url: string) {
-        const headers = new HttpHeaders().set('Content-Type', 'application/json').set('authorization', 'Eevoo0aemah1ohY6Oheehee4ivahR5ae');
-        RestService.processingRequest = true;
-        console.log(globalThis.wallet);
-        return lastValueFrom(this.httpClient
-            .post(url, globalThis.wallet, {headers: headers}))
-            .then(res => this.processResponse(res))
-            .catch(this.handleError);
     }
 
-    public signTx(signature: any, data: any) {
+    public signAndFinalizeTx(signature: any, data: any) {
         const headers = new HttpHeaders().set('Content-Type', 'application/json').set('authorization', 'Eevoo0aemah1ohY6Oheehee4ivahR5ae');
-        const url = '/rwd/multisig/finalize/6/sporwc/' + data.id;
+        const url = '/rwdbuild/multisig/finalize/1/sporwc/' + data.id;
```

Comment:
Do it here like with the other endpoints end replace 1 with "customerId" then we do not have any hard coded stuff inside. 
Even if we for now just use a global variable we can than be sure that its changing at all sport until we have a good strategy for environment variables

Reply:
Added two globals in the navbar

        globalThis.customerId = 1;
        globalThis.multiSigType = "sporwc";